integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -160,6 +160,16 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
     }
   }
 
+  private void waitForFilePersisted(long fileId) {
```

Comment:
This is an integration test, so we shouldn't be mocking something as important as the file system worker. I will create PrivateAccess harnesses similar to https://github.com/amplab/tachyon/blob/master/servers/src/test/java/tachyon/master/file/meta/TtlBucketPrivateAccess.java